### DISCUSSION TITLE
naiveproxy 与 caddy-trojan 配置问题

### DISCUSSION THREAD

**Discussion by gsgqgyq** [lxhao61/integrated-examples#31] [Q&A] [answered]
感谢配置示例，解决了我的大问题！有两个疑问：

1. naiveproxy 代理必须使用 naiveproxy 客户端吗？
   我以前使用 caddy-forwardproxy 插件可以直接借用 SwitchyOmega 插件使用https代理。现在尝试了大佬你所编译的，发现无法连接。
    > 本示例 naiveproxy 除了支持 HTTP/2 代理应用，还同时支持 HTTP/3 代理应用，即 QUIC 协议传输。
  
    这必须借用 naiveproxy 客户端才能做到？
    
2.  caddy-trojan 的 WebSocket 设置？
    ```
          {
            "handle": [{
              "handler": "trojan",
              "connect_method": true,
              "websocket": true
            }]
          }, //此部分配置为caddy-trojan插件的WebSocket应用，若删除就仅支持Trojan应用。
	```
	
	这个想要使用 CDN，不需要设置 path 吗？
再次感谢大佬的分享！

**Comment by lxhao61** [comment, marked as answer]
1、是的。naiveproxy采用的是改进版forwardproxy插件，不是原版forwardproxy插件。
2、是的。caddy-trojan插件目前不支持Websocket应用设置path 。
